### DISCUSSION TITLE
Prisma client output bug

### DISCUSSION THREAD

**Discussion by simibruda** [prisma/prisma#24870] [Q&A] [answered]
### Question

I have a problem with my Prisma setup. I have two schemas and I want to generate two clients. The issue arises when I run yarn dev. It creates the dist folder without the generated folder, resulting in the following error:

php
Copy code
at Module._compile (node:internal/modules/cjs/loader:1504:14)
at Module._extensions..js (node:internal/modules/cjs/loader:1588:10)
at Module.load (node:internal/modules/cjs/loader:1282:32) {
  code: 'MODULE_NOT_FOUND',
}
Has anyone encountered this issue or can provide guidance on how to resolve it? I use Nest js 

![Screenshot 2024-07-24 at 10 40 09](https://github.com/user-attachments/assets/50f39d0a-ebb7-4bce-ba3f-cae8a1f987a7)
![Screenshot 2024-07-24 at 10 40 22](https://github.com/user-attachments/assets/45445ee3-8159-4c5f-8db4-2434a4969f77)

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment, marked as answer]
Hi @simibruda 👋 

Thank you for raising this question.

How did you define your `postinstall` script? Does it look like the below:
```
// Make sure your postinstall script in package.json correctly generates both Prisma clients. 

"postinstall": "prisma generate --schema prisma/schema1.prisma && prisma generate --schema prisma/schema2.prisma"

```
Are you also correctly importing the Prisma Client? You will need to change the import path from

```js
import { PrismaClient } from '@prisma/client'

to

import { PrismaClient } from '../prisma/generated/xxxx' where xxxx represents any path you're using.
```
You may also want to take a look at this workaround described in this similar [issue](https://github.com/prisma/prisma/issues/21066#issuecomment-1727797786).
